src/object/cli_obj.c

```diff
@@ -1546,6 +1969,30 @@ dc_obj_update(tse_task_t *task)
 		goto out_task;
 	}
 
+	struct daos_oclass_attr *oca =
+				daos_oclass_attr_find(obj->cob_md.omd_id);
```

Comment:
please define this with other variable at the beginning of the function. thanks.

Reply:
A (bad?) habit from years of C++ programming... I will fix.